ubcpi/static/html/ubcpi.html

```diff
@@ -214,31 +227,35 @@ <h2 id="correct-option-head" ng-if="options.length == rc.correct_answer && rc.co
             <div class="ubcpi-class-breakdown">
 
                 <a name="classbreakdown"></a>
-                <h3 class="ubcpi-class-breakdown-head">Class Breakdown</h3>
+                <h5 class="ubcpi-class-breakdown-head">Class Breakdown</h5>
                 <p>This is a look at how your classmates answered the question during the initial and final rounds.</p>
-                <h4>Answer Options</h4>
+                <h6>Answer Options</h6>
                 <div class="ubcpi-breakdown-answer-options">
 
                     <p class="ubcpi-option" data-ng-repeat="option in options track by $index">
                         <img ng-src="{{option.image_url}}" id="original-option-image-{{ $index }}" alt="{{option.image_alt}}" ng-if="option.image_position == 'above' && option.image_url" />
 
                         <span class="ubcpi-breakdown-option-num"><i>Option {{$index + 1}}</i></span>&nbsp;<span class="ubcpi-breakdown-correct-option" ng-if="rc.correct_answer === $index" style="font-style:italic">(correct)</span>
                         <span ng-if="rc.correct_answer === $index" class="ubcpi-breakdown-answer-text ubcpi-show-correct">{{option.text}}</span>
+                        <span ng-if="rc.correct_answer === $index" class="sr">{{rc.calc($index)}}</span>
                         <span ng-if="rc.correct_answer !== $index" class="ubcpi-breakdown-answer-text">{{option.text}}</span>
+                        <span ng-if="rc.correct_answer !== $index" class="sr">{{rc.calc($index)}}</span>
 
                         <img ng-src="{{option.image_url}}" id="original-option-image-{{ $index }}" alt="{{option.image_alt}}" ng-if="option.image_position == 'below' && option.image_url" />
 
                         <span class="chosen-option option-details-text" ng-if="rc.answer_original == $index && rc.status() == rc.ALL_STATUS.ANSWERED">(You chose this option)</span>
                     </p>
 
                 </div>
-                <h4>Initial Answer Selection</h4>
+
+                <h6>Initial Answer Selection</h6>
                 <div class="original-answers results-container choicegroup">
 
                     <pi-barchart options="options" stats="rc.stats.original" correct="rc.correct_answer" role="user_role"></pi-barchart>
```

Comment:
As stated in the original review, this is invalid HTML.  role takes an enumerated value. see the [HTML5 spec](https://www.w3.org/TR/html5/dom.html#aria-role-attribute)
> The attribute, if specified, must have a value that is a set of space-separated tokens representing the various WAI-ARIA roles that the element belongs to.

HTML5 specified data-* attributes for this.  If Angular needs this specified, you can use data-role.